app/javascript/src/components/Dashboard/Notes/NoteTable.jsx

```diff
@@ -56,14 +114,64 @@ export default function NoteTable({
               </td>
               <td>
                 <div className="flex flex-row items-center justify-start text-gray-900">
-                  {note.title}
+                  <a href="#">{note.title}</a>
```

Comment:
Please use neetoUI Button with `style="link"`.
Refer the documentation [here](https://neeto-ui.netlify.app/#/button)